nrf_modem/doc/sockets.rst

```diff
@@ -476,6 +476,38 @@ In this case, either security tag 4 or security tag 5 can be used for operations
 
    Using multiple security tags
 
+TLS traffic decryption
+**********************
+
+Starting from modem firmware v2.0.0, Transport Layer Security (TLS) traffic can be decrypted with Nordic tools if the TLS session is created using certificates stored to security tags ranging from :c:macro:`nrf_sec_tag_tls_decrypt_0` to :c:macro:`nrf_sec_tag_tls_decrypt_19`.
+
+.. important::
+   These security tags shall be used only for test and development purposes.
+
+Testing TLS traffic decryption 
+------------------------------
+
+Prerequisites:
+
+#. Make sure that the device is running an application that utilizes TLS. The TLS session shall be created using certificates stored to tags ranging from :c:macro:`nrf_sec_tag_tls_decrypt_0` to :c:macro:`nrf_sec_tag_tls_decrypt_19`.
+#. Make sure that the application has modem traces enabled. Information on modem traces and how to enable them is found in the :ref:`modem_trace` documentation.
+#. Make sure that the device is running modem firmware v2.0.0 and higher. Information on how to find the modem firmware version is found in the `Revision Identification +CGMR <https://docs.nordicsemi.com/bundle/ref_at_commands/page/REF/at_commands/general/cgmr_set.html>`_ documentation.
+#. Make sure that Wireshark is installed on your machine. Information on Wireshark and how to install it is found at the `Wireshark <https://www.wireshark.org/>`_ website.
+
+Test TLS traffic decryption by performing the following steps:
+
+#. |connect_kit|
+#. Open the `Cellular Monitor`_ desktop application and connect the device.
+#. Select :guilabel:`Autoselect` from the **Modem trace database** drop-down menu, or a modem firmware version that is programmed on the device.
+#. Deselect :guilabel:`Refresh dashboard on start`.
```

Comment:
Why does this need to be disabled?

Reply:
Actually, it doesn't, at least explicitly. As far as I've seen it gets disabled automatically when selecting `Open in Wireshark`.
Keep in mind that there are different versions of the Cellular Monitor app and some GUI items are not present in later versions. Not sure yet how and if this should be approached from documentation standpoint.